media-gfx/openvdb/openvdb-9.0.0-r3.ebuild

```diff
@@ -14,16 +14,18 @@ SRC_URI="https://github.com/AcademySoftwareFoundation/${PN}/archive/v${PV}.tar.g
 LICENSE="MPL-2.0"
 SLOT="0/9"
 KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~x86"
-IUSE="cpu_flags_x86_avx cpu_flags_x86_sse4_2 +blosc doc +nanovdb numpy python static-libs test utils zlib abi6-compat abi7-compat abi8-compat +abi9-compat"
+IUSE="cpu_flags_x86_avx cpu_flags_x86_sse4_2 +blosc cuda doc +nanovdb numpy python static-libs test utils zlib abi6-compat abi7-compat abi8-compat +abi9-compat"
 RESTRICT="!test? ( test )"
 
 REQUIRED_USE="
-	numpy? ( python )
 	^^ ( abi6-compat abi7-compat abi8-compat abi9-compat )
+	cuda? ( nanovdb )
+	numpy? ( python )
 	python? ( ${PYTHON_REQUIRED_USE} )
 "
+
 RDEPEND="
-	>=dev-cpp/tbb-2021.4.0:=
+	>=dev-cpp/tbb-2020.3
```

Comment:
dropped :=

Reply:
2020.3 doesn't have subslots

Reply:
It has an implicit 0/0 which then, when upgrading to a newer version, is a change (e.g. 0/2021 or whatever), so it matters if we know a newer version has them, as we want to get rebuilt when the ABI changes

Reply:
i see